sandbox/animation.js

```diff
@@ -45,8 +45,8 @@ dropdownBtn.addEventListener("click", function() {
 });
 
 function selectCurrentGroup(group, index, animation) {
-    if (currentGroupIndex !== undefined) {
-        document.getElementById(formatId(currentGroup.name + "-" + currentGroupIndex)).classList.remove("active");
+    for (var i = 0; i < dropdownContent.children.length; i++) {

```

Comment:
why?

Reply:
i had a bug on this but can't reproduce it now.
I revert to the update of current option.

Reply:
done